packages/terra-file-path/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-file-path",
+  "main": "lib/FilePath.js",
+  "version": "0.1.0",
+  "description": "Terra File Path is a horizontal menu that indicates where the user is. It is also more commonly known as a breadcrumb component.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git",
+    "directory": "packages/terra-file-path"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-file-path",
+    "file-path",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": ">=2.8.0, <6.0.0"
```

Comment:
Can we change the format for clarity?
```suggestion
    "react-intl": "^2.8.0 || 5"
```

Reply:
Wouldn't versions 3, and 4 be excluded with the new format?

Reply:
According to https://semver.npmjs.com, both of these are identical:

**Before:**
<img width="880" alt="CleanShot 2023-09-01 at 15 02 31@2x" src="https://github.com/cerner/terra-framework/assets/38024451/7473527c-dfee-4e85-87f6-ee441e487abf">

**After:**
<img width="879" alt="CleanShot 2023-09-01 at 15 01 17@2x" src="https://github.com/cerner/terra-framework/assets/38024451/7376b78f-b148-455d-9490-0734e69c2820">

(if anything, it considered the comma in the existing syntax to be incorrect)

Reply:
[6bc9e07](https://github.com/cerner/terra-framework/pull/1771/commits/6bc9e0793b1875c8d9188e6f9360e7a6b27f2719)